Make ROCm Install more explicit in README + info about 2.0 release

bact opened the pull request:
- Fix Docker link
- Make command line style consistent with original TensorFlow README (use $ to mark prompt)
- Make sections to make the ROCm installation instructions more visible
- Add info about TensorFlow 2.0 ROCm port release

bact committed:
Make ROCm Install more explicit in README

- Fix docker link
- Make command line style consistent with original TensorFlow README (use $ to mark prompt)
- Make sections to make the ROCm installation instructions more visible

ROCm-Apps-Test commented:
Can one of the admins verify this patch?

ROCm-Apps-Test commented:
Can one of the admins verify this patch?

ROCm-Apps-Test commented:
Can one of the admins verify this patch?

ROCm-Apps-Test commented:
Can one of the admins verify this patch?

bact committed:
Update with info about TensorFlow ROCm 2.0

jerryyin commented on README.md:
```diff
@@ -30,25 +30,49 @@ to
+see [ROCm-specific README file](README.ROCm.md).
+
+- TensorFlow ROCm 1.14 works with ROCm2.6.
+- TensorFlow ROCm 2.0 works with ROCm2.9.
```
It sounds like a good idea to document the past ROCm compatible version here. Though we should document TF ROCm 1.15 because it will be the long lasting v1 release.

Will need @sunway513 to decide on the update on this main docs page.

jerryyin commented on README.md:
```diff
@@ -30,25 +30,49 @@ to
-**Tensorflow ROCm port**
-This project is based on TensorFlow 1.14.0. It has been verified to work with the latest ROCm2.6 release.
-Please follow the instructions [here](https://github.com/RadeonOpenCompute/ROCm-docker/blob/master/quick-start.md) to set up your ROCm stack.
-A docker container: **rocm/tensorflow:latest(https://hub.docker.com/r/rocm/tensorflow/)** is readily available to be used:
```
Since this points to the specific container, maybe use the tags page instead https://hub.docker.com/r/rocm/tensorflow/tags

bact committed:
Update README.md

Use https://hub.docker.com/r/rocm/tensorflow/tags link to point to `rocm/tensorflow:latest`
